docs/swupd-add-pkg.4.rst

```diff
@@ -0,0 +1,48 @@
+=================
+swupd-add-pkg.sh
+=================
+
+------------------------------------
+Adds custom package for swupd to use
+------------------------------------
+
+:Copyright: \(C) 2017 Intel Corporation, CC-BY-SA-3.0
+:Manual section: 4
+
+
+SYNOPSIS
+========
+
+``swupd-add-pkg.sh``
+
+``/usr/bin/swupd-add-pkg.sh``
+
+
+DESCRIPTION
+===========
+
+Adds a given local package to the specified <bundle_name> file, and
+automatically generates all content with mixer for swupd to use. It creates
+a very minimalist mix, unlike regular mixer usage where one composes the
+entire OS. This instead outputs only the new bundle content, and os-core -
+to signify the version changes - in order to augment the regular upstream
+update content. Swupd is aware of this new user added content, and if it
+exists on the system, swupd will attempt to use it for any of its operations,
+e.g update, verify, bundle add, etc.
```

Comment:
Packages cannot ever conflict or mismatch between bundles, otherwise update creation (chroots actually) will fail. Adding the package would error out and the user would have to fix the conflict before they would ever get update content for it, so this is OK.

Reply:
I think we determined swupd should error out in the case of collisions unless an override is passed. That should be added to the documentation.

Reply:
the `--allow-mix-collisions` should still be documented.